contrib/core-contract-tests/tests/pox-4/pox_StackAggregationCommitSigCommand.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  logCommand,
+  PoxCommand,
+  Real,
+  Stub,
+  Wallet,
+} from "./pox_CommandModel.ts";
+import { Pox4SignatureTopic, poxAddressToTuple } from "@stacks/stacking";
+import { expect } from "vitest";
+import { Cl } from "@stacks/transactions";
+import { bufferFromHex } from "@stacks/transactions/dist/cl";
+
+/**
+ * The `StackAggregationCommitSigCommand` allows an operator commits partially
```

Comment:
Same NIT as above:

Change from: "allows an operator commits partially..."

To: "allows an operator to partially commit stacked STX & to allocate..."

Reply:
Commit txs do commit partially stacked stx. No change required.